### PR TITLE
Draft linking Net to main doc.

### PR DESCRIPTION
**Pull request by maximelb**
- Adding to the Table of Content
- Adding to the main page.

Note that the png of lc-net on the main page does not resolve to any, but @tekgrunt mentioned that he has a logo ready to go.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -41,6 +41,19 @@ By default LimaCharlie stores a full year of searchable endpoint telemetry as we
 * [Artifact Collection](./lc-storage.md#artifact-collection)
 * [Historical Threat Hunting](./lc-storage.md#historical-threat-hunting)
 
+<img src="https://storage.googleapis.com/lc-edu/content/images/logos/lc-net-i.png"
```
Comment (amrikr) [inline docs/index.md:44]: This URL doesn't seem right.

Comment (maximelb) [reply docs/index.md:44]: Yeah, I just adjusted it from the other elements on the page, see note in my PR ^

Comment (amrikr) [reply docs/index.md:44]: Ahhh, that's what I get for immediately jumping to the review!

**Review by amrikr — APPROVED** [review]
👍🏽

**Pull request merged**